CHANGES.md

```diff
@@ -1,3 +1,39 @@
+# snare 0.2.0 (xxxx-xx-xx)
+
+## Breaking changes
+
+* The `github`-block level `reposdir` option has been removed. The more
+  flexible `match`-block level `cmd` has been introduced. In essence:
+
+    ```
+    github {
+      reposdir = "/path/to/prps";
+      ...
+    }
+    ```
+
+  should be changed to:
+
+    ```
+    github {
+      match ".*" {
+        cmd = "/path/to/reposdir/%o/%r %e %j";
+      }
+    }
+    ```
+
+  in order to obtain the previous behaviour. `snare` informs users whose config
```

Comment:
I'd remove "in order to obtain the previous behaviour". I think that's obvious.

Reply:
Agreed. Fixed in https://github.com/softdevteam/snare/pull/43/commits/a78d6dc9e4a200132d4f1a6cfa736a9226ffefbb.

Reply:
> `snare` informs users whose config contains `repodir` how to update it.

The grammar here feels somehow clumsy.

How about:

> Upon encountering `repodir`, `snare` will now print a message to help the user to migrate.

Reply:
I must admit that I don't like the current phrasing very much, but I think making it long-winded makes it even more clumsy :/ I'm not sure it's worth wasting too much time over TBH.

Reply:
Let me distill out what I dislike about the sentence. Eliding the middle part of the sentence, we have:

> snare informs users how to update it

Sounds a bit cave man.

> > snare informs users **on** how to update it

Seems to make it better. So does this work perhaps?
> snare informs users whose config contains repodir **on** how to update it.

If not, agreed, let's move on.

Reply:
`on` doesn't feel grammatically correct either :( While I agree the current phrasing sucks, I think we should move on.

Reply:
yep